frontend/components/events/create/DescriptionRichTextEditor.tsx

```diff
@@ -18,22 +12,16 @@ export default function DescriptionRichTextEditor({
 }) {
   const editor = useEditor({
     editable: true,
-    extensions: [
-      StarterKit,
-      Underline,
-      Link,
-      Superscript,
-      SubScript,
-      Highlight,
-      TextAlign.configure({ types: ["heading", "paragraph"] }),
-      Placeholder.configure({ placeholder: "Write your description in here!" }),
-    ],
+    extensions: [StarterKit, Link],
```

Comment:
why are we removing all these extensions?

Reply:
added them back, i think when i did it for smaller screens i thought we didnt need them lol